lib/tasks/evm_ansible_runner.rake

```diff
@@ -1,10 +1,11 @@
-require 'awesome_spawn'
-require "vmdb/plugins"
-
 namespace :evm do
   namespace :ansible_runner do
     desc "Seed galaxy roles for provider playbooks"
     task :seed do
+      require 'awesome_spawn'
+      require "vmdb/plugins"
+      require 'ansible/content'
```

Comment:
Just to be clear:  This is the only line change that is required.

The rest is just my pedantic self getting the best of me...